src/playback/config.js

```diff
@@ -19,35 +19,219 @@ const SIZES = [
+  {id: "widevine-vp9-sfr-chrome", variant: "widevine-vp9-sfr", browser: "chrome", suite: "raptor-youtube-playback-chrome-live"},
+
+  {id: "hfr-geckoview", variant: "hfr", browser: "geckoview", suite: "raptor-youtube-playback-geckoview-live"},
+  {id: "av1-sfr-geckoview", variant: "av1-sfr", browser: "geckoview", suite: "raptor-youtube-playback-av1-sfr-geckoview-live"},
+  {id: "h264-sfr-geckoview", variant: "h264-sfr", browser: "geckoview", suite: "raptor-youtube-playback-h264-sfr-geckoview-live"},
+  {id: "vp9-sfr-geckoview", variant: "vp9-sfr", browser: "geckoview", suite: "raptor-youtube-playback-vp9-sfr-geckoview-live"},
+  {id: "widevine-hfr-geckoview", variant: "widevine-hfr", browser: "geckoview", suite: "raptor-youtube-playback-widevine-hfr-geckoview-live"},
+  {id: "widevine-h264-sfr-geckoview", variant: "widevine-h264-sfr", browser: "geckoview", suite: "raptor-youtube-playback-geckoview-live"},
+  {id: "widevine-vp9-sfr-geckoview", variant: "widevine-vp9-sfr", browser: "geckoview", suite: "raptor-youtube-playback-geckoview-live"},
+
+  {id: "hfr-fenix", variant: "hfr", browser: "fenix", suite: "raptor-youtube-playback-fenix-live"},
+  {id: "av1-sfr-fenix", variant: "av1-sfr", browser: "fenix", suite: "raptor-youtube-playback-av1-sfr-fenix-live"},
+  {id: "h264-sfr-fenix", variant: "h264-sfr", browser: "fenix", suite: "raptor-youtube-playback-h264-sfr-fenix-live"},
+  {id: "vp9-sfr-fenix", variant: "vp9-sfr", browser: "fenix", suite: "raptor-youtube-playback-vp9-sfr-fenix-live"},
+  {id: "widevine-hfr-fenix", variant: "widevine-hfr", browser: "fenix", suite: "raptor-youtube-playback-widevine-hfr-fenix-live"},
+  {id: "widevine-h264-sfr-fenix", variant: "widevine-h264-sfr", browser: "fenix", suite: "raptor-youtube-playback-fenix-live"},
+  {id: "widevine-vp9-sfr-fenix", variant: "widevine-vp9-sfr", browser: "fenix", suite: "raptor-youtube-playback-fenix-live"},
+
+  {id: "hfr-fennec68", variant: "hfr", browser: "fennec68", suite: "raptor-youtube-playback-fennec68-live"},
+  {id: "av1-sfr-fennec68", variant: "av1-sfr", browser: "fennec68", suite: "raptor-youtube-playback-av1-sfr-fennec68-live"},
+  {id: "h264-sfr-fennec68", variant: "h264-sfr", browser: "fennec68", suite: "raptor-youtube-playback-h264-sfr-fennec68-live"},
+  {id: "vp9-sfr-fennec68", variant: "vp9-sfr", browser: "fennec68", suite: "raptor-youtube-playback-vp9-sfr-fennec68-live"},
+  {id: "widevine-hfr-fennec68", variant: "widevine-hfr", browser: "fennec68", suite: "raptor-youtube-playback-widevine-hfr-fennec68-live"},
+  {id: "widevine-h264-sfr-fennec68", variant: "widevine-h264-sfr", browser: "fennec68", suite: "raptor-youtube-playback-fennec68-live"},
+  {id: "widevine-vp9-sfr-fennec68", variant: "widevine-vp9-sfr", browser: "fennec68", suite: "raptor-youtube-playback-fennec68-live"},
+];
+
+const HFR_H264_TESTS = selectFrom(HFR_SIZES.slice(0,2))
```

Comment:
`.slice(0,2)` is a mystery.  Instead of 

    selectFrom(HFR_SIZES.slice(0,2))

try

    selectFrom(HFR_SIZES)
        .limit(2)  // SOME DESCRIPTION OF WHY